src/main/scripts/setup-glassfish.py

```diff
@@ -5,7 +5,10 @@
 import platform
 import shlex
 import subprocess
-import io
+try:
+    import io
+except ImportError:
+    import StringIO as io
 import getpass
```

Comment:
This should not be neccessary. The `io` module is in the [Python standard lib](https://docs.python.org/2.7/library/io.html) since Python 2.6 and thus this `ImportError` will never be raised.